Description: Use debian/dirs instead of install-data-local
 Replace ugly upstream hack with debian/dirs
 .
 swapspace (1.12-1) experimental; urgency=low
 .  
   * New Maintainer (Closes: #725821)
   * New upstream
   * Account for tmpfs (Closes: #691128)
   * Redo packaging to use new upstream build system
Author: Jacob Adams <tookmund@gmail.com>  

---

--- a/Makefile.am
+++ b/Makefile.am
@@ -1,5 +1,7 @@
 SUBDIRS = src
 sysconf_DATA = swapspace.conf
 
-install-data-local:
-	$(MKDIR_P) $(localstatedir)/lib/swapspace
+#Hack uneccessary in package; using debian/dirs instead
+#install-data-local:
+#	$(MKDIR_P) $(localstatedir)/lib/swapspace
+
